tools/editor/project_manager.cpp

```diff
@@ -871,8 +871,8 @@ ProjectManager::ProjectManager() {
 
 	HBoxContainer *top_hb = memnew( HBoxContainer);
 	vb->add_child(top_hb);
-	TextureFrame *logo = memnew( TextureFrame );
-	logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
+	//TextureFrame *logo = memnew( TextureFrame );
+	//logo->set_texture(theme->get_icon("LogoSmall","EditorIcons"));
```

Comment:
Yeah teaching @reduz to *let it go* (please don't sing!) when it comes to obsoleted code is a long process :P